app/src/main/java/com/internship/move/presentation/menu/MenuAccountFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.internship.move.presentation.menu
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.internship.move.R
+import com.internship.move.databinding.FragmentMenuAccountBinding
+import com.internship.move.presentation.menu.viewmodel.MenuViewModel
+import com.zhuinden.fragmentviewbindingdelegatekt.viewBinding
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class MenuAccountFragment : Fragment(R.layout.fragment_menu_account) {
+
+    private val binding by viewBinding(FragmentMenuAccountBinding::bind)
+    private val viewModel by viewModel<MenuViewModel>()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        viewModel.getUserRequest("Bearer ${viewModel.getLoginToken()}")
+
+        initListeners()
+        initObserver()
+
+    }
+
+    private fun initListeners() {
+        binding.logoutIV.setOnClickListener {
```

Comment:
I prefer leaving an empty line between setters (same for the observers). It makes the code a bit clearer imo